src/vm/i386/unixstubs.cpp

```diff
@@ -6,7 +6,7 @@
 
 extern "C"
 {
-    void ThrowControlForThread()
+    void ThrowControlForThread(ONWIN64EXCEPTIONS(FaultingExceptionFrame *pfef))
```

Comment:
It would be better to revise this once we could run "Hello, World" with WIN64EXCEPTIONS.